documentation/testing/test-plans/20240717_release-test-plan.md

```diff
@@ -0,0 +1,61 @@
+﻿# Release Test Plan
+Release date: 17/07/2024
+
+### Introduction
+The purpose of this test plan is to outline the approach and scope for testing the updates made as part of this release. It will detail the various types of testing that will be conducted as part of release process.  This plan aims to ensure all aspects of the updates are thoroughly tested and meet the required standards before deployment.
+
+### Scope 
+This test plan covers the functional and sanity testing to validate all features and bug fixes included in the release.
+
+### Test Strategy
+#### Types of Testing
+- Manual Functional Testing
+- Sanity Testing
+
+### Approach
+#### Manual functional Testing: 
+Manual functional testing will be carried out to validate fixes and updates are in place in pre-prod. 
+#### Sanity Testing
+This testing will be carried out in pre-prod to ensure all functionalities are working as expected. 
+
+### Test Scope
+#### Issues to be Tested:
```

Comment:
This doesn't list all the issues. WAF changes included.

Reply:
updated it.